intuita-webview/src/jobDiffView/DiffViewer/Collapsable.tsx

```diff
@@ -0,0 +1,65 @@
+import { forwardRef, useImperativeHandle, useState } from 'react';
+import { ReactComponent as ArrowDownIcon } from '../../assets/arrow-down.svg';
+import './Collapsable.css';
+
+type CollapsableProps = {
+	defaultExpanded: boolean;
+	headerComponent: React.ReactNode;
+	headerClassName?: string;
+	children: React.ReactNode;
+	contentClassName?: string;
+	className?: string;
+};
+
+export type CollapsableRefMethods = {
+	expanded: boolean;
+	collapse: () => void;
+	expand: () => void;
+};
+
+export const Collapsable = forwardRef<CollapsableRefMethods, CollapsableProps>(
+	(
+		{
+			defaultExpanded: defaultCollapsed,
+			headerComponent,
+			headerClassName,
+			contentClassName,
+			className,
+			children,
+		},
+		ref,
+	) => {
+		const [expanded, setExpanded] = useState(defaultCollapsed);
+
+		useImperativeHandle(ref, () => ({
+			expanded,
+			collapse: () => setExpanded(false),
+			expand: () => setExpanded(true),
+		}));
+
+		return (
+			<div className={`collapsable ${className ?? ''} `}>
+				<div
+					className={`collapsable__header ${headerClassName ?? ''} `}
```

Comment:
can you use `classnames` lib here?

Reply:
@henoktsegayetoptal 's part
Benny: addressed it